tests/test_utils.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime
+
+from app.main.utils import calculate_days_to_deadline
+from config import Config
+
+
+def test_default_behaviour(flask_test_client):
+    mock_current_date = datetime.strptime("20/09/2023", "%d/%m/%Y").date()
+
+    days_remaining = calculate_days_to_deadline(mock_current_date)
+
+    assert days_remaining == 56
```

Comment:
I think Noel was advising to not pass a `current_date` and to just use the default param to ensure it behaves as expected. We won't be able to assert a particular int value but we can still then assert that we get an int type back